client/src/components/GreenIcon/GreenIcon.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import './GreenIcon.scss';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+class GreenIcon extends Component {
```

Comment:
I don't think you'll need separate components for "GreenIcon" and "RedIcon" to change the color. You could add a color property to `Icon` and pass in the color directly from there.